datumaro/components/dataset.py

```diff
@@ -751,13 +784,27 @@ def transform(self, method: Union[str, Type[Transform]],
             raise TypeError("Unexpected 'method' argument type: %s" % \
                 type(method))
 
-        self._data.transform(method, *args, **kwargs)
+        self._data.transform(method, **kwargs)
         if self.is_eager:
             self.init_cache()
 
         return self
 
     def run_model(self, model, batch_size=1) -> Dataset:
+        """
+        Applies a model to dataset items' media and produces a dataset with
+        media and annotations.
+
+        Args:
+            model (Launcher or Modeltransform) - The model to be
```

Comment:
These types ought to be in the function signature (also, `ModelTransform` is misspelled).

Reply:
Fixed